gremlin-javascript/src/main/javascript/gremlin-javascript/test/integration/client-behavior-tests.js

```diff
@@ -63,6 +63,33 @@ describe('Client', function () {
 
             await noUserAgentClient.close();
         });
+        it('should not request permessage deflate compression by default', async function () {
+            let result = await client.submit('1', null, {requestId: settings.SEC_WEBSOCKET_EXTENSIONS});
```

Comment:
for immutable variables better to use [const](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const)
same below